components/Nav/Nav.test.tsx

```diff
@@ -125,4 +125,22 @@ describe('Nav', () => {
 
     expect(mockPush).toHaveBeenCalledWith('/login');
   });
+
+  it('it should close the modal when the signout button is clicked', async () => {
+    mockUsePathname.mockImplementation(() => '/weeklyPicks');
+
+    render(<Nav />);
+
+    const drawerTrigger = screen.getByTestId('drawer-trigger');
+
+    fireEvent.click(drawerTrigger);
+
+    const signOutButton = screen.getByTestId('sign-out-button');
```

Comment:
nit: it feels like this is lacking organization. Is there it does not follow a structure such as

```tsx
it('it should close the modal when the signout button is clicked', async () => {
	const drawerTrigger = screen.getByTestId('drawer-trigger'); 
	const signOutButton = screen.getByTestId('sign-out-button');

	mockUsePathname.mockImplementation(() => '/weeklyPicks');

	render (<Nav />);

	fireEvent.click(drawerTrigger);
	fireEvent.click(signOutButton);

	await waitFor(() => {
		expect(drawerTrigger.getAttribute('data-state')).toBe('closed');
    });
  });